OpenRA.Mods.Common/Traits/AffectsShroud.cs

```diff
@@ -33,9 +33,12 @@ public abstract class AffectsShroud : ConditionalTrait<AffectsShroudInfo>, ITick
 
 		readonly HashSet<PPos> footprint;
 
-		[Sync] CPos cachedLocation;
-		[Sync] WDist cachedRange;
-		[Sync] protected bool CachedTraitDisabled { get; private set; }
+		[Sync]
+		CPos cachedLocation;
+		[Sync]
+		WDist cachedRange;
+		[Sync]
+		protected bool CachedTraitDisabled { get; private set; }
```

Comment:
Same here, putting an empty line between them would be more consistent with the rest of the PR.